ibm/mas_devops/roles/uds_install/tasks/main.yml

```diff
@@ -111,3 +111,153 @@
     - uds_contact.first_name is defined and uds_contact.first_name != ""
     - uds_contact.last_name is defined and uds_contact.last_name != ""
     - uds_contact.email is defined and uds_contact.email != ""
+
+
+
+# 9. Generate properties used by gencfg_uds
```

Comment:
This whole section from here to the end should all be in the new gencfg_uds role rather than the install role. Also the task above (number 8) doesn't exist as you deleted it.